.deploy/init.sh

```diff
@@ -7,13 +7,15 @@ if [ ! -e /docker_build ]; then
   exit 1
 fi
 
+export DEBIAN_FRONTEND=noninteractive
+
 echo "** Installing apt dependencies"
 # This is done by the dockerfile, but the intermediate issue can be cached, so do
 # it again here.
 apt-get clean
 apt-get update -y
 
-apt-get install -y ruby php-cli zip unzip locales
+LC_ALL=C apt-get install -y ruby php-cli zip unzip locales
```

Comment:
vscode executes stuff a bit differently, including implicitly copying locale around. Installing the `locales` package will fail if the locale is set to `en_US.UTF-8` before we have ran `locale-gen en_US.UTF-8`... which we can't do earlier as it's provided by the locales package.

Reply:
Shall we move `apt-get install` to the `Dockerfile`, to allow docker caching across Codespaces?

Reply:
This script is used by a few different dockerfiles at the moment, though it's probalby worth splitting the stuff that doesn't require a copy of the source into a separate script which can then be called from the dockerfile and cache

Reply:
... which won't work as that script isn't available until the source is mounted.